RaidBot/BusinessLogic/Raids/RaidService.cs

```diff
@@ -526,16 +537,98 @@ public class RaidService {
          return EmbedBuilderHelper.RaidEmbedBuilder("All current raids:", raids);
       }
 
+      public ModuleResult ChangeAttendanceMode(string raidName, string attendanceMode, IUser requesterUser, IUser userToUpdate) {
+         var result = new ModuleResult();
+         var raids = _raidFileService.GetRaidsFromFile().Where(a => a.Name.Equals(raidName, StringComparison.CurrentCultureIgnoreCase));
+
+         if (raids.Count() == 1) {
+            string[] validModes = { "inperson", "in-person", "remote" };

```

Comment:
Since there are only 2 options, I think it could be better to not specify what you are changing to and just write `$mode raid1`

Reply:
True, it's much simpler this way! Just updated it.